Amplify/AmplifyDevMenu/Amplify+DevMenu.swift

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+extension Amplify {
+
+    @available(iOS 13.0.0, *)
+    public static var devMenu: AmplifyDevMenu?
+
+    static var isDeveloperMenuEnabled = false
+
+    @available(iOS 13.0.0, *)
+    public static func enableDevMenu(devMenuDelegate: DevMenuDelegate) {
+        devMenu = AmplifyDevMenu(delegate: devMenuDelegate)
+        isDeveloperMenuEnabled = true
+    }
+
+    static func isDevMenuEnabled() -> Bool {
+        return isDeveloperMenuEnabled
```

Comment:
Do we need an extra variable to check if enabled? Can we just check if deMenu == nil